package.json

```diff
@@ -32,7 +32,7 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "flow": "flow",
-    "devserver": "nodemon ./lib/backend/server.js"
+    "devserver": "NODE_ENV=development nodemon ./lib/backend/server.js"
```

Comment:
um, if you say so

Reply:
To clarify on that I don't run the npm devserver with the pm2 process, I just do `node ./lib/backend/server.js` for that

devserver is just when we do our stuff locally.